AISKU/Tests/Selenium/Tests.html

```diff
@@ -20,6 +19,7 @@
         var modules = new ModuleLoader({
             baseUrl: '../../',
             paths: {
+                sinon: "./node_modules/sinon/pkg/sinon",
```

Comment:
Changed so it now uses the package.json sinon version rather than always v1.17.2